meson.build

```diff
@@ -50,8 +52,14 @@ else
   
   if dxvk_compiler.get_id() == 'msvc'
     lib_d3dcompiler_47 = dxvk_compiler.find_library('d3dcompiler')
+    wrc = [ find_program('windres'), '-i', '@INPUT@', '-o', '@OUTPUT@' ]
   else
     lib_d3dcompiler_47 = dxvk_compiler.find_library('d3dcompiler_47')
+    if (cpu_family == 'x86_64')
+      wrc = [ find_program('x86_64-w64-mingw32-windres'), '-i', '@INPUT@', '-o', '@OUTPUT@' ]
+    else
+      wrc = [ find_program('i686-w64-mingw32-windres'), '-i', '@INPUT@', '-o', '@OUTPUT@' ]
```

Comment:
Can this be cleaned up so that we have **one** definition of `wrc`? There's no reason to have 3x `find_program` and the entire parameter list in the code.